include/inviwo/qt/editor/inviwomainwindow.h

```diff
@@ -113,18 +115,23 @@ class IVW_QTEDITOR_API InviwoMainWindow : public QMainWindow, public NetworkEdit
     bool askToSaveWorkspaceChanges();
     void exitInviwo(bool saveIfModified = true);
     void showAboutBox();
+    
+    void showWelcomeScreen();
+    void hideWelcomeScreen();
 
 protected:
     virtual void dragEnterEvent(QDragEnterEvent* event) override;
     virtual void dragMoveEvent(QDragMoveEvent* event) override;
     virtual void dropEvent(QDropEvent* event) override;
 
 private:
+    friend WelcomeWidget;
```

Comment:
Remove?

Reply:
either `WelcomeWidget` is a friend of InviwoMainWindow, or we need to make 
`bool openWorkspace(QString workspaceFileName, bool exampleWorkspace);`
and `QStringList getRecentWorkspaceList() const;` public.

Reply:
`openWorkspace(QString, bool)`  should not necessarily be exposed in my opinion.

Reply:
make them public, thats way better then friends....